MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: post
+title: Customization in .NET MAUI Barcodes (SfBarcodeGenerator) | Syncfusion
+description: Learn here all about Customization of Syncfusion .NET MAUI Barcodes(SfBarcodeGenerator) control with key features and more.
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# Customization in .NET MAUI Barcodes (SfBarcodeGenerator)
+
+## Text customization
+
+### Displaying input value
+
+The provided input value of the barcode can be displayed by enabling its `ShowText` property. By default, the `ShowText` is false.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="4" %}
+
+        <barcode:SfBarcodeGenerator Value="12634388927" 
+                                    HeightRequest="150"
+                                    WidthRequest="300" 
+                                    ShowText="True"/>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Customization Show Text](images/customization/maui-customization-show-text.png)
+
+### Text style customization
+
+The style of the text can be customized with the `TextStyle` property of the barcode generator.
```

Comment:
The style of the text can be customized using the `TextStyle` property of the barcode generator.

Reply:
Changes updated.